src/module.common.ts

```diff
@@ -0,0 +1,141 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
```

Comment:
It looks like Material `2.0.0-alpha.9` fixes this and only requires one import now.